cmd/detect.go

```diff
@@ -75,6 +75,16 @@ func runDetect(cmd *cobra.Command, args []string) {
 		detector.AddGitleaksIgnore(filepath.Join(source, ".gitleaksignore"))
 	}
 
+	// ignore findings from the baseline (an existing report in json format generated earlier)
+	baselinePath, _ := cmd.Flags().GetString("baseline-path")
+	if baselinePath != "" {
+		detector.AddBaseline(baselinePath)
```

Comment:
@Gawan-0e278eed I think you're missing assigning the `err` here.

Reply:
Added in line 81. Regarding line 78, I believe it is fine to ignore any error here. If gitleaks doesn't get a baseline path it can just continue without any baselining.